Userland/Libraries/LibGUI/Event.h

```diff
@@ -63,6 +63,7 @@ class Event : public Core::Event {
         WM_WindowIconBitmapChanged,
         WM_AppletAreaSizeChanged,
         WM_SuperKeyPressed,
+        WM_SuperSpaceKeyPressed,
```

Comment:
@SpencerCDixon Alfred provides Hotkey preference.
Assistant could also have configuration to let user choose specific keys.

This would require Assistant to register those keys with Taskbar as trigger key.

Also a persistent preference/configuration can be used for more use cases

Reply:
Yeah that would be really cool to add one day! Talked to Kling about that initially and decided to keep it simple with hard coded action to start with idea of adding a generalized hotkey system in the future potentially.